magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -207,12 +184,34 @@ def mc_dl0_to_dl1(input_file, output_dir, config):
 
     zenith = 90 - sim_config["max_alt"].to_value("deg")
     azimuth = Angle(sim_config["max_az"]).wrap_at("360 deg").degree
-
+    logger.info(np.asarray(list(assigned_tel_ids.values())))
+    LSTs_IDs = np.asarray(list(assigned_tel_ids.values())[0:4])
+    LSTs_in_use = np.where(LSTs_IDs > 0)[0] + 1   #Here we select which LSTs are/is in use
+    
+    if len(LSTs_in_use) == 0:
+        LSTs_in_use = ''.join(str(k) for k in LSTs_in_use)
+    elif len(LSTs_in_use) > 0:
+        LSTs_in_use = 'LST'+'_LST'.join(str(k) for k in LSTs_in_use)
+        
+    MAGICs_IDs = np.asarray(list(assigned_tel_ids.values())[4:6])
+    MAGICs_in_use = np.where(MAGICs_IDs > 0)[0] + 1    #Here we select which MAGICs are/is in use
+    
+    if len(MAGICs_in_use) == 0:
+        MAGICs_in_use = ''.join(str(k) for k in MAGICs_in_use)
+    elif len(MAGICs_in_use) > 0:
+        MAGICs_in_use = 'MAGIC'+'_MAGIC'.join(str(k) for k in MAGICs_in_use)
+    magic_clean = {}
+    for k in MAGICs_IDs:
+        if k > 0:           
+            magic_clean[k] = MAGICClean(camera_geoms[k], config_magic["magic_clean"])
+    
     output_file = (
         f"{output_dir}/dl1_{particle_type}_zd_{zenith.round(3)}deg_"
-        f"az_{azimuth.round(3)}deg_LST-1_MAGIC_run{obs_id}.h5"
-    )
+        f"az_{azimuth.round(3)}deg_{LSTs_in_use}_{MAGICs_in_use}_run{obs_id}.h5"
+    )                                                                                 #The files are saved with the names of all telescopes involved
```

Comment:
you could drop the large number of spaces here and the two empty lines below

Reply:
Done